notebooks/generate_synthetic_data.ipynb

```diff
@@ -429,20 +390,101 @@
+    "        # Stories are scraped using this script: \n",
+    "        # https://github.com/EEdwardsA/data-mining-script-for-ao3\n",
+    "        \n",
+    "        s = random.randint(0, 200)\n",
+    "        story = open(f'./stories/story{s}.txt')\n",
+    "        \n",
+    "        # Turn story into string object\n",
+    "        text = story.read()\n",
+    "        \n",
+    "        # Randomly assign width\n",
+    "        width = random.randint(500, 900)\n",
+    "        width_scaler = random.randint(16,20)\n",
+    "        font_size = int((width)/width_scaler)\n",
+    "        height = int((len(text)*font_size)/25)\n",
+    "        \n",
+    "        # \"Pencil\" darkness\n",
+    "        c = random.randint(0, 96)\n",
+    "                 \n",
+    "        img = Image.new('RGB', (width, height), color = random.choice(colors))\n",
+    "        draw = ImageDraw.Draw(img)\n",
+    "\n",
+    "        # Create font object with a font file and specify\n",
+    "        # desired size\n",
+    "\n",
+    "        font = ImageFont.truetype(random.choice(fonts), size=font_size)\n",
+    "\n",
+    "        # Starting position of story is fixed\n",
+    "\n",
+    "        (x, y) = (50, 50) \n",
+    "        \n",
+    "        color = f'rgb({c}, {c}, {c})' # Grayscale colors to imitate pencil\n",
+    "\n",
+    "        # Draw the message on the background\n",
+    "\n",
+    "        draw.text((x, y), \"\\n\".join(textwrap.wrap(text, \n",
+    "                                                  width=int(width/width_scaler)) \n",
+    "                                                #width=int(6*width/font_size)\n",
+    "                                                 ), \n",
+    "                                    fill=color, font=font, features='aalt') #features = 'aalt' should let you use\n",
+    "                                                                            # fonts that have multiple symbols per character,\n",
+    "                                                                            # but you will need to get libraqm working.\n",
+    "                                                                            # It's supposed to come packaged with Pillow 5.0+\n",
+    "        \n",
+    "        # Half of the samples pages will have lines on them\n",
+    "        \n",
+    "        # offset = random.randint(-5,5) # Randomize where text is relative to line\n",
+    "        offset = 0\n",
+    "        \n",
+    "        flip = random.randint(0,1)\n",
+    "        if flip:\n",
+    "            pass\n",
+    "#         draw.line((0,50+font_size+offset, width,50+font_size+offset), fill='black')\n",
+    "        for j in range(50+font_size+offset, height, int(height/(font_size+offset))):\n",
+    "            draw.line((0, j, width, j), fill='black')\n",
+    "        \n",
+    "        img.save(f'./synth_images/img{i}','jpeg')\n"
    ]
```

Comment:
naming these recommendations and clarifying the why behind them are important transfers of knowledge to the incoming team